sdk/core/azure-core/inc/azure/core/operation_status.hpp

```diff
@@ -9,6 +9,8 @@
 
 #pragma once
 
+#include "azure/core/export.hpp"
+
```

Comment:
nit: Is this necessary here? Why?
```suggestion
```

Reply:
Yes, definitions below use it, as we discussed offline